vs code extension/client/package.json

```diff
@@ -3,7 +3,7 @@
 	"description": "Cobol-check extension to run cobol unit tests",
 	"author": "Open Mainframe Project",
 	"license": "MIT",
-	"version": "0.3.2",
+	"version": "0.3.3",
```

Comment:
It is package.json in the root of the extension folder, not inside the client or server folders